### DISCUSSION TITLE
[Feature] Auto close issue if no further comments received

### DISCUSSION THREAD

**Discussion by oriongonza** [community/community#108474] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Add a button that schedules an issue for closing in n days if no further comments received. This is useful because maintainers often times close issues too fast to avoid them piling on. You can always re open them but that's not great.

Depending on the issue you could auto close in 5 days, 1 month, 2 years...

This would be useful for bug reports too, if more information is required and the issuer never shows up again.
If they show up after it's been auto closed it can always be reopened but at least it's something automatic and it doesn't give off the vibe "I don't care about your bug, closing your issue".

**Comment by DJLink** [comment, marked as answer]
You can configure actions to auto-close inactive issues

https://docs.github.com/en/actions/managing-issues-and-pull-requests/closing-inactive-issues
